docs/book/how-to/customize-docker-builds/use-a-prebuilt-image.md

```diff
@@ -0,0 +1,114 @@
+# Skip building an image on every pipeline run
+
+When running a pipeline on a remote Stack, ZenML builds a Docker image with a base ZenML image and adds all of your project dependencies and your pipeline code to it. This process might take significant time depending on how big your dependencies are, how powerful your local system is and how fast your internet connection is (to pull base layers and push the final image to your container registry). 
```

Comment:
This is wrong right because zenml by default only builds the image if it detects environment changes

Reply:
correct! i added some text stressing that this doesn't always happen